src/main/java/org/influxdb/dto/QueryResult.java

```diff
@@ -69,6 +71,9 @@ public void setError(final String error) {
   }
 
   public static class Result {
+    @Json(name = "statement_id") private long statementId = 0;
```

Comment:
with primitives, default is `0` for integers and `false` for boolean so, just remove this please.